index.bs

```diff
@@ -467,6 +515,35 @@ http://school.register.gov.uk/religious-character/Quaker.json
 </pre>
 </div>
 
+## Records proof resource ## {#records-proof-resource}
+
+<dl class="resource"><dt>Path</dt><dd>/proof/records/{proof-identifier}</dd></dt></dl>
```

Comment:
might be worth introducing this earlier in the document (ie before record-proof-resource) so that you don't have forward-references to it?